### ISSUE TITLE
Can't alt/tab on OnBoard after close of window

### ISSUE THREAD

**Issue by DuMuT6p** [linuxmint/cinnamon#8897]
```
 * Cinnamon version - 4.0.10
 * Distribution - Mint 19.1
 * Graphics hardware *and* driver used - Nvidia proprietary driver
 * 64 bit
 ```

**Issue**
Can't alt/tab switch windows. It seems OnBoard holds the focus, but don't register the key combination.

**Steps to reproduce**
1. Open OSK
2. Open several windows (doesn't matter program or folder)
3. Close one 

**Expected behaviour**
Should be able to alt/tab.

**Comment by icarter09** [comment]
Just to make sure I'm tracking correctly....is the issue that you are closing the OSK, then not able to alt/tab since the focus is still being held by the OSK?

**Comment by DuMuT6p** [comment]
@icarter09 As far as I can tell without debugging - yes, that should be the issue.

**Comment by icarter09** [comment]
@DuMuT6p thanks. I'm not able to reproduce this issue. I opened the OSK, pressed alt/tab and several other buttons, closed out the OSK, and was able to still use alt/tab on my regular keyboard. The focus was picked back up by my regular keyboard. I tried this several times and was not able to reproduce the issue.

**Comment by DuMuT6p** [comment]
OK. Sorry. I meant I am closing another window(e.g. folder in nemo) and the OSK catches the focus and blocks alt+tab. I suppose catching the focus is normal even if not the immediate "after", but it should not block.

**Comment by icarter09** [comment]
@DuMuT6p I'm still not able to reproduce the issue are you having. My two guesses for this is that I'm using Cinnamon 4.2.4 instead of 4.0.10. Another guess is that you are using the OSK for a longer session than just the one or two things (like me during debugging). I've noticed you have put in several tickets related to the OSK, which is a good thing :)

**Comment by DuMuT6p** [comment]
@icarter09 its Mint 19.1 not 19.2. Very sorry for confusion. It could also be the Cinnamon version. As for (2), all issues are logged with clean start of OSK.
I mean to update to 19.2 soon so I will retest all issues then.

**Comment by icarter09** [comment]
@DuMuT6p thanks for the update and no worries on the confusion.

**Comment by DuMuT6p** [comment]
I upgraded to 19.2 and updated with all available updates as of today. Issue still persist. 

I also noticed that it reproduces for closing of any kind. I was extracing an archive and when the progress window auto closed, it was not possible to alt/tab.

**Comment by icarter09** [comment]
@DuMuT6p are you still experiencing this issue?

**Comment by DuMuT6p** [comment]
@icarter09 yes. I have upgraded to 19.3 / Cinnamon 4.4.8 and still the same issue. If I make a clean reinstall I will retest and update.

**Comment by RoboBurned** [comment]
I can confirm this issue in fresh installation of Linux Mint 19.3 Cinnamon 4.4.8.
After I close a folder or any application like terminal by using [x] in a corner or by Alt + F4 and then press Alt+Tab then nothing happens.
